perllib/FixMyStreet/App/Controller/Admin/Reports.pm

```diff
@@ -234,6 +234,18 @@ sub edit : Path('/admin/report_edit') : Args(1) {
             push @fields, { name => 'Duplicates', val => join( ',', @{ $problem->get_extra_metadata('duplicates') } ) };
             delete $extra->{duplicates};
         }
+
+        if ( $extra->{contributed_by} ) {
+            my $u = $c->cobrand->users->find({id => $extra->{contributed_by}});
+            push @fields, { name => 'Created By', val => "@{[$u->name]} (@{[$u->email]})" };
```

Comment:
The HTML escaping might make this more trouble than it's worth, but could this link to the the admin user search results for their email to make it easier to find their other reports or view their timeline for example?

Reply:
The descriptions also need marking for translation in some way.